model_compression_toolkit/core/pytorch/reader/graph_builders.py

```diff
@@ -41,8 +40,8 @@ def _extract_parameters_and_buffers(module: Union[torch.nn.Module, GraphModule],
         Dictionary containing module parameters and buffers by name.
     """
 
-    named_parameters = {name: to_numpy(parameter) for name, parameter in module.named_parameters()}
-    named_buffers = {name: to_numpy(buffer) for name, buffer in module.named_buffers()}
+    named_parameters = {name: parameter for name, parameter in module.named_parameters()}
```

Comment:
Can you explain why you removed numpy conversion? seems like something that might have effect on different modules, and is not directly related to reused layers